.github/workflows/acceptance-tests.yml

```diff
@@ -407,21 +22,15 @@ jobs:
         uses: actions/setup-go@v4
         with:
           go-version-file: 'go.mod'
+      - name: Install Imposter
```

Comment:
I modified this file to test the action in this PR as a new action must be merged into master to run it in PRs. `mock-acceptace-tests.yml` will be the workflow that uses the mock server. This workflow will remain the same.